src/use-idle/use-idle.ts

```diff
@@ -0,0 +1,73 @@
+import { IdleController } from './idle-controller'
+
+const defaultEvents = ['mousemove', 'mousedown', 'resize', 'keydown', 'touchstart', 'wheel'];
+const oneMinute = 60e3;
+
+export const useIdle = (controller: IdleController, ms: number = oneMinute, initialState: boolean = false, events: string[] = defaultEvents) => {
+  let isIdle = initialState;
+  let timeout = setTimeout(() => {
+    isIdle = true
+    dispatchAway();
+  }, ms);
+
+  const method = (methodName: string): Function => {
```

Comment:
In case this branch gets merged after #22, I have extracted this function into a helper function share between all modules. But don't worry about this for now

Reply:
Yeah I've seen that, makes sense 👍